native/src/components/EventsDateFilter.tsx

```diff
@@ -60,7 +60,7 @@ const EventsDateFilter = ({
   setEndDate,
   modalOpen,
   setModalOpen,
-}: EventsDateFilterProps): JSX.Element => {
+}: EventsDateFilterProps): ReactElement => {
```

Comment:
The global `JSX` namespace is being removed, and we use `ReactElement` almost everywhere https://react.dev/blog/2024/04/25/react-19-upgrade-guide#the-jsx-namespace-in-typescript